project.clj

```diff
@@ -47,7 +47,6 @@
                  [org.yaml/snakeyaml]
                  [commons-lang]
                  [commons-io]
-                 [bidi "1.23.1"]
```

Comment:
We talked in person about this, but we'll need to keep the bidi override in place to get to 2.1.3 otherwise clj-parent will win.

Reply:
Updated